src/components/pages/photos/elements/roverNavbar/RoverNavbar.js

```diff
@@ -2,12 +2,12 @@ import React from 'react';
 import RoverButton from './RoverButton';
 import './RoverNavbar.css';
 
-export default function RoverNavbar() {
+export default function RoverNavbar(props) {
   return (
     <div className="rover-navbar">
-      <RoverButton roverName="Curiosity" />
-      <RoverButton roverName="Opportunity" />
-      <RoverButton roverName="Spirit" />
+      <RoverButton roverName="Curiosity" onClick={() => props.onClick('Curiosity')} />
+      <RoverButton roverName="Opportunity" onClick={() => props.onClick('Opportunity')} />
+      <RoverButton roverName="Spirit" onClick={() => props.onClick('Spirit')} />
```

Comment:
I notice we're using these rover name strings a lot in the codebase - maybe we should [store the strings in an enum somewhere](https://www.sohamkamani.com/blog/2017/08/21/enums-in-javascript/#enumerations-with-objects)?